src/components/Button/Button.test.jsx

```diff
@@ -28,65 +28,91 @@ describe('Button', () => {
     });
   });
 
+  describe('Sizes', () => {
+    const sizes = [
+      { name: 'Small', class: 'sm' },
+      { name: 'Medium', class: '' },
+      { name: 'Large', class: 'lg' },
+    ];
+
+    sizes.map(size => (
+      describe(`${size.name}`, () => {
+        test(`it has a ${size.class} class applied to it`, () => {
+          render(<Button size={size.class}>{`${size.name} Button`}</Button>);
+
+          const btn = screen.getByText(`${size.name} Button`).closest('button');
+
+          expect(btn.getAttribute('class')).toContain(size.class);
+        });
+      })
+    ));
+  });
```

Comment:
This was recommended by Code Climate, through an inline PR comment. Hooray for robots!

Reply:
Heck yes! @JCarmine this is similar to how we wrote the `Heading` tests. I love it.